gateway/exporters/fluentd/fluentd.go

```diff
@@ -0,0 +1,229 @@
+package fluentd
+
+import (
+	"errors"
+	"fmt"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/fluent/fluent-logger-golang/fluent"
+	"github.com/openconfig/gnmi-gateway/gateway/configuration"
+	"github.com/openconfig/gnmi-gateway/gateway/connections"
+	"github.com/openconfig/gnmi-gateway/gateway/exporters"
+	"github.com/openconfig/gnmi-gateway/gateway/utils"
+	"github.com/openconfig/gnmi/ctree"
+	gnmipb "github.com/openconfig/gnmi/proto/gnmi"
+)
+
+const Name = "fluentd"
+const MetricType = "log"
+const LogType = "log"
+const LoggedMetricType = "loggedMetric"
+
+var _ exporters.Exporter = new(FluentdExporter)
+
+type Log struct {
```

Comment:
nit: I'd rename this to LogEntry. Log might imply a collection of log messages or a logger interface.